Realm/RLMRealm.mm

```diff
@@ -180,6 +185,77 @@ + (instancetype)realmWithURL:(NSURL *)fileURL {
     configuration.fileURL = fileURL;
     return [RLMRealm realmWithConfiguration:configuration error:nil];
 }
+
++ (dispatch_queue_t)asyncOpenDispatchQueue {
+    static dispatch_queue_t queue;
```

Comment:
This can just be `static dispatch_queue_t queue = dispatch_queue_create("io.realm.asyncOpenDispatchQueue", DISPATCH_QUEUE_CONCURRENT); return queue;`.

Or just forgo creating the queue entirely.